### PR TITLE
Near the vote button show the end date of the election

### PR DESCRIPTION
**Pull request by pypingou**
Fixes https://github.com/fedora-infra/nuancier/issues/79

### CODE REVIEW

File: nuancier/templates/index.html [hunk 1]
```diff
@@ -46,6 +46,7 @@
         <a class="large_button" href="{{ url_for('election', election_id=election.id) }}">
             {{ election.election_name }} - {{ election.election_year }}
         </a>
+        until {{ election.election_date_start }}
```
Comment (ralphbean) [inline nuancier/templates/index.html]: Shouldn't this be the ``end`` date?  not the ``start`` date?

Comment (pypingou) [inline nuancier/templates/index.html]: Most likely, will double check, but most likely :)

**Comment by pypingou** [comment]
You were right :)

**Comment by ralphbean** [comment]
Cool, :+1:

**Pull request merged**